src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1113,14 +1130,32 @@ def setup(self):
 
         self.output = FlaxBigBirdSelfOutput(self.config, dtype=self.dtype)
 
-    # Copied from transformers.models.bert.modeling_flax_bert.FlaxBertAttention.__call__ with Bert->BigBird
-    def __call__(self, hidden_states, attention_mask=None, deterministic=True, output_attentions: bool = False):
+    def __call__(
+        self,
+        hidden_states,
+        attention_mask,
+        layer_head_mask=None,
+        deterministic=True,
+        output_attentions: bool = False,
+    ):
         # Attention mask comes in as attention_mask.shape == (*batch_sizes, kv_length)
         # FLAX expects: attention_mask.shape == (*batch_sizes, 1, 1, kv_length) such that it is broadcastable
         # with attn_weights.shape == (*batch_sizes, num_heads, q_length, kv_length)
-        attn_outputs = self.self(
-            hidden_states, attention_mask, deterministic=deterministic, output_attentions=output_attentions
-        )
+        if self.config.attention_type == "original_full":
+            attn_outputs = self.self(
+                hidden_states,
+                attention_mask,
+                layer_head_mask=layer_head_mask,
+                deterministic=deterministic,
+                output_attentions=output_attentions,
+            )
+        else:
+            attn_outputs = self.self(
+                hidden_states,
+                attention_mask,
+                deterministic=deterministic,
+                output_attentions=output_attentions,
+            )
```

Comment:
why don't we use the head_mask here?

Reply:
Actually, head masking is not used for this type of attention in `torch` implementation either. However, I think this should be somehow handled as well. @vasudevgupta7 - I know you contributed `BigBird` to `transformers`. Do you have any idea, how this should be handled?